tests/waku_keystore/utils.nim

```diff
@@ -0,0 +1,31 @@
+{.used.}
+
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
```

Comment:
I don't think we need this in our files anymore :)
cc: @Ivansete-status for confirmation

Reply:
Exactly @rymnc , only ` {.push raises: [].}`

Reply:
Alrighty :)